server/auth/types/openid/routes.ts

```diff
@@ -127,7 +127,15 @@ export class OpenIdAuthRoutes {
             state: nonce,
             scope: this.openIdAuthConfig.scope,
           };
+          if (this.config.openid?.additional_parameters) {
+            for (const [key, value] of Object.entries(this.config.openid?.additional_parameters)) {
+              console.log(`${key}: ${value}`);
+              query[key] = value;
```

Comment:
Nit: This would allow overriding other query parameters; however, I think this would be relatively obvious since the config declaration is side by side.

Reply:
I've added the following warning just in case

![image](https://github.com/opensearch-project/security-dashboards-plugin/assets/128482925/15e4f793-c69d-4db8-a8ac-84ae752949dc)

Reply:
This PR looks good to me, but I don't believe it should be able to overwrite the required fields for the OIDC token endpoint. The fields from [here](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/auth/types/openid/routes.ts#L165-L175) should not be able to be overwritten by the additional parameters. This [RFC](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.3) should contain the required fields needed in the access token request that should not be able to be overwritten by additional params.